pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -243,10 +243,11 @@
         },
 
         "ubuntu": {
-            "#import": [ "linux" ]
+            "#import": [ "debian" ]
         },
+
         "ubuntu-x64": {
-            "#import": [ "ubuntu", "linux-x64" ]
+            "#import": [ "ubuntu", "debian-x64" ]
         },
 
         "ubuntu.14.04": {
```

Comment:
If there is a specific version compatibility you could do the same at those versions.  The purpose of the version-less identifiers is to represent assets that are not tied to any particular version of the OS.